src/service/network.ts

```diff
@@ -221,6 +247,8 @@ export class Network extends Service {
             'wired': ['jsobject'],
             'primary': ['string'],
             'connectivity': ['string'],
+            'ipv4-address': ['string'],
+            'ipv6-address': ['string'],
```

Comment:
why is this here?

Reply:
Removed in the next commit, it was just an idea to make it one place where you can collect info, by accident i pushed this